src/plugins/condition/utils/operations.js

```diff
@@ -76,7 +76,7 @@ export const OPERATIONS = [
     {
         name: 'between',
         operation: function (input) {
-            return input[0] > parseInt(input[1]) && input[0] < parseInt(input[2]);
+            return Number(input[0]) > Number(input[1]) && Number(input[0]) < Number(input[2]);
```

Comment:
wondering need to handle case where user enters large number first and small number as second value.

Reply:
Sure I can do that.

Reply:
Done.